src/View/html/FormSettings/list.php

```diff
@@ -34,7 +34,7 @@
 				<div class="tablenav top">
 					<div class="alignleft actions bulkactions"></div>
 					<div class="alignright">
-						<a class="button" href="<?= esc_url( $args['add_new_url'] ) ?>" aria-label="<?= esc_attr__( 'Add new PDF', 'gravity-forms-pdf-extended' ) ?>"><?php esc_html_e( 'Add New', 'gravityforms' ); ?></a>
+						<a class="button" href="<?= esc_url( esc_url( $args['add_new_url'] ) ) ?>" aria-label="<?= esc_attr__( 'Add new PDF', 'gravity-forms-pdf-extended' ) ?>"><?php esc_html_e( 'Add New', 'gravityforms' ); ?></a>
```

Comment:
@jestonihpi you've wrapped the output in `esc_url()` twice. One of these will need to be removed.